CMakeLists.txt

```
@@ -49,6 +66,13 @@ if(DEFINED CMAKE_JS_VERSION)
     include(NodeJSTargets)
 endif()
 
+if (NOT MSVC)
+    # Ninja buffers output so we need to tell the compiler to use colors even though stdout isn't a tty.
+    if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
```

Is there any reason this needs to be wrapped in a string template?
```suggestion
    if(CMAKE_GENERATOR STREQUAL "Ninja")
```